CharlieBackend.Panel/HtmlHelpers/CalendarHelper.cs

```diff
@@ -11,6 +11,24 @@ namespace CharlieBackend.Panel.HtmlHelpers
 {
     public static class CalendarHelper
     {
+        public static HtmlString CalendarHeaderHtml(this IHtmlHelper html, CalendarDisplayType displayType)
+        {
+            string result = string.Empty;
+
+            foreach (DayOfWeek day in Enum.GetValues(typeof(DayOfWeek)))
+            {
+                TagBuilder div = GetDayBlock(day, displayType);
+
+                div.InnerHtml.Append(day.ToString());
+
+                using var writer = new System.IO.StringWriter();
+                div.WriteTo(writer, HtmlEncoder.Default);
+                result += writer.ToString();
```

Comment:
suggest use stringbuilder instead string, of course it`s not a very long loop but anyway

Reply:
Done!